packages/grafana-ui/src/components/Layout/Stack/Stack.tsx

```diff
@@ -40,11 +40,12 @@ interface StackProps extends Omit<React.HTMLAttributes<HTMLElement>, 'className'
   direction?: ResponsiveProp<Direction>;
   wrap?: ResponsiveProp<Wrap>;
   children?: React.ReactNode;
+  flexGrow?: ResponsiveProp<CSSProperties['flexGrow']>;
```

Comment:
A small comment on this, in `Box` we have `grow?: ResponsiveProp<number>;`, which assigns something to the prop `flex` and not to `flexGrow`:
```
      getResponsiveStyle(theme, grow, (val) => ({
        flex: val,
      })),
```

The reason I made it set `flex` instead of `flexGrow` was because of the note at the end of [this block](https://css-tricks.com/snippets/css/a-guide-to-flexbox/#aa-flex). By setting `flex` to a number it also sets `flexBasis` to 0% which is an easy way to make them all the same size.

We can decide to do either, but this should be consistent with `Box` both in terms of name and of type. Wdyt?

Reply:
Good point, agree, except that maybe we'd name the prop `flex`, as `grow` is a bit misleading in this case?

Reply:
But `flex` would then need to accept all of that properties' options, like `flex: 1 1 0%`, etc. Alternatively we could make the `Box` one also work on `flexGrow` instead of `flex`? Not sure if this changes how any of the places that use `Box` at the moment look like

Reply:
I'm just thinking that if we only add the `grow` prop, eventually we'll get requests to add the other ones 😄 Also feels a bit weird not to include the whole set of flex props. Should we add `grow`, `shrink` and `basis` props to both `Box` and `Stack`? I think this would solve most of the issues and will make the components more future-proof.

Reply:
I'll merge this as is and we can update the API later.